### PR TITLE
Prex

### PR DESCRIPTION
**Pull request by paigeasaurusrex**
Made a lot of formatting changes and created the svg for the legend. Having trouble with the color legend package that's needed to make the legend because I think it references "d3" which we have rendered as "Null"? LMK if you have any thoughts/ideas how to handle this!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
+    <script>d3version6 = d3
+        window.d3 = null</script>
     <script src="https://d3js.org/d3.v3.js"></script>
-    <script>
-        d3version3 = d3
-        window.d3 = null
-    </script>
+    <script>d3version3 = d3
+        window.d3 = null</script>
+    <!--need to import this script in order for the legend to work, but I think the code above makes it not work because we use
```
Comment (maharte) [inline index.html:12]: Depending on how big the js file is, could we just add it to the repo and find/replace?

File: index.html [hunk 2]
```diff
@@ -2,17 +2,17 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>World Happiness Report</title>
+    <title>World Happiness Report Test</title>
```
Comment (maharte) [inline index.html:5]: Is this intentional?

File: index.html [hunk 3]
```diff
@@ -97,16 +106,15 @@
...
         <div class="viz-item" style="flex-shrink: 0">
             <svg id="bar-plot"></svg>
         </div>
     </div>
 
 
     <script src="javascript/scoreboard_filters.js" type="application/javascript"></script>
-    <script>
-        const files = [];
+    <script>const files = [];
```
Comment (maharte) [inline index.html:117]: spacing is weird here

File: index.html (hunk 2 above)
Comment (paigeasaurusrex) [reply index.html:5]: changing this back

File: index.html (hunk 3 above)
Comment (paigeasaurusrex) [reply index.html:117]: Not sure what/why that changed? Changing it back

**Pull request merged**